config/config.yaml

```diff
@@ -209,7 +209,9 @@ branch-protection:
           branches:
             master:
               protect: true
-        template-repository:
+        plugins:
+          required_pull_request_reviews:
+            required_approving_review_count: 2
```

Comment:
```suggestion
```

Since it's an incubating project, we could stick with only 1 approval at the moment.

Reply:
Ok, fixed.

Reply:
Thank you!